docs/SUMMARY.md

```diff
@@ -54,4 +54,6 @@
 
 # Misc
 
-- [Misc](./misc/README.md)
+- [Overview](./misc/README.md)
+- [State Sync Dump](./misc/state_sync_dump.md)
```

Comment:
This file appears to be missing?

Reply:
hmm it's this one right? https://github.com/near/nearcore/blob/master/docs/misc/state_sync_dump.md. At least when i run `mdbook build` that page shows up. It just wasn't hooked up to the table on the left of the page before